e2e/suites/actions-available/files-folders/single-file-actions.test.ts

```diff
@@ -48,26 +49,31 @@ describe('Files - available actions : ', () => {
   const { searchInput } = page.header;
   const searchResultsPage = new SearchResultsPage();
 
-  beforeAll(async () => {
-    await adminApiActions.login();
-    await adminApiActions.createUser({ username });
-    await userActions.login(username, username);
+  beforeAll(async (done) => {
+    try {
+      await adminApiActions.login();
+      await adminApiActions.createUser({ username });
+      await userActions.login(username, username);
```

Comment:
i don't see userActions being used here, so this login might not be needed.
same for the other files as well

Reply:
I went through all the classes and tried to remove the login were not needed. Could you, please, have a look again?
Thanks